ui/packages/catalog-ui-search/src/main/webapp/js/cql.js

```diff
@@ -19,6 +19,9 @@
 // jshint ignore: start
 const moment = require('moment')
 
+const ANYTEXT_WILDCARD = '"anyText" ILIKE \'%\''
+const ANYTEXT_EMPTY = '"anyText" ILIKE \'\''
```

Comment:
:question: Should empty return different results than wildcard?

Reply:
I'm not positive. I was told there was a difference between empty and wildcard, but I can play around a bit and see if that's true.